### PR TITLE
Tweaking

### PR DESCRIPTION
**Pull request by mantognini**
a few improvements

### CODE REVIEW

File: config/bundles/bluima.commons_0.1.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "1.0.1",
   "dependencies": [
     {
-      "value": "ch.epfl.bbp.nlp:bluima_commons:1.0.1"
+      "value": "ch.epfl.bbp.nlp:bluima_commons:1.0.2-SNAPSHOT"
```
Comment (renaud) [inline config/bundles/bluima.commons_0.1.json]: why SNAPSHOT? Didi you need to modify anything in `bluima_commons`?

**Comment by renaud** [comment]
Thanks Marco!

File: config/bundles/bluima.commons_0.1.json (hunk 1 above)
Comment (mantognini) [inline config/bundles/bluima.commons_0.1.json]: Yes, I did change the case of "configFile" parameter (https://github.com/BlueBrain/bluima/commit/28a63939c4b52ef57c2a5eb4db3615ddcbfcad88) as you asked. ;-)

**Comment by mantognini** [comment]
You're welcome.

**Pull request merged**